include/ola/client/ClientTypes.h

```diff
@@ -29,6 +29,8 @@
 #include <string>
 #include <vector>
 
+#include "common/protocol/Ola.pb.h"
```

Comment:
Please don't include this here as it exposes the internal rpc messages to the callers.

I'm fine if you want to make a

ObjectFactory {
  static OlaPlugin FromProtobuf(ola::proto::PluginInfo plugin_info);
  static OlaInputPort FromProtobuf(ola::proto::PortInfo port_info);
  etc. 
}

and keep ObjectFactory.h out of the install path.